specification/notebooks/resource-manager/Microsoft.Notebooks/preview/2019-10-11-preview/notebooks.json

```diff
@@ -0,0 +1,227 @@
+            "200": { 
+              "description": "Success" 
+            }, 
+            "204": {
+              "description": "Success"
+            } 
+          }
+        }
+      },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Notebooks/NotebookProxies/{resourceName}/updatePrivateLinkId/{privateLinkId}": {
+        "post": {
+          "tags": [ "NotebookResource" ],
+          "operationId": "PostPrivateLinkIdNotebookResource",
+          "consumes": [ "application/json-patch+json", "application/json", "text/json", "application/*+json" ],
+          "produces": [ "application/json-patch+json", "application/json", "text/json", "application/*+json" ],
+          "parameters": [
+            {
+                "name": "subscriptionId",
+                "in": "path",
+                "required": true,
+                "type": "string"
+            },
+            {
+                "name": "resourceGroupName",
+                "in": "path",
+                "required": true,
+                "type": "string"
+            },
+            {
+                "name": "resourceName",
+                "in": "path",
+                "required": true,
+                "type": "string"
+            },
+            {
+                "name": "privateLinkId",
+                "in": "path",
+                "required": true,
+                "type": "string"
+            }
+          ],
+          "responses": { "200": { "description": "Success" } }
+        }
+      },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Notebooks/NotebookProxies/{resourceName}/removePrivateLinkId/{privateLinkId}": {
+        "post": {
+          "tags": [ "NotebookResource" ],
+          "operationId": "RemovePrivateLinkIdNotebookResource",
+          "consumes": [ "application/json-patch+json", "application/json", "text/json", "application/*+json" ],
+          "produces": [ "application/json-patch+json", "application/json", "text/json", "application/*+json" ],
+          "parameters": [
+            {
+                "name": "subscriptionId",
+                "in": "path",
+                "required": true,
+                "type": "string"
+            },
+            {
+                "name": "resourceGroupName",
+                "in": "path",
+                "required": true,
+                "type": "string"
+            },
+            {
+                "name": "resourceName",
+                "in": "path",
+                "required": true,
+                "type": "string"
+            }
+          ],
+          "responses": { "200": { "description": "Success" }
+        }
+      }
+    },
+    "definitions": {
+      "NotebookResource": {
+        "type": "object",
+        "properties": {
+          "location": { "type": "string" },
+          "properties": { "$ref": "#/definitions/NotebookResourceProperties" },
+          "hostname": { "type": "string" }
```

Comment:
this cant be a top level property, it must be moved under "properties" property

Reply:
Fixed